desk/lib/dm.hoon

```diff
@@ -44,7 +44,7 @@
     =/  parent   (get id)
     ?~  parent   [sum threads]
     ?:  ?=(%| -.writ.u.parent)  [sum threads]
-    =*  writ  +.writ.u.parent
```

Comment:
For these, where the lark is the very last thing being addressed, this is a meaningful change. In this context (and the `=,`s in /app/chat) it doesn't matter in practice, since all you do with the result is pull parts out of it, but worth keeping an eye out.